pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -1025,6 +1032,25 @@ spec:
 ---
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
+metadata:
+  name: jiva-volume-delete-listreplicapod-default
+spec:
+  meta: |
+    id: deletelistreppods
+    runNamespace: {{ .Volume.runNamespace }}
+    disable: {{ .Config.RetainReplicaData.enabled }}
+    apiVersion: v1
+    kind: Pod
+    action: list
+    options: |-
+      labelSelector: openebs.io/replica=jiva-replica,openebs.io/persistent-volume={{ .Volume.owner }}
+    retry: "12,10s"
```

Comment:
`retry` is not required. This is useful if post has any `verify` related specifications?
In other words, this is retry on verification error.

Reply:
ref - https://github.com/openebs/maya/blob/master/pkg/task/task.go#L223